src/NServiceBus.Core/Routing/AutomaticSubscriptions/AutoSubscribe.cs

```diff
@@ -43,7 +43,7 @@ protected internal override void Setup(FeatureConfigurationContext context)
 
                     var messageTypesHandled = GetMessageTypesHandledByThisEndpoint(handlerRegistry, conventions, settings);
 
-                    return new ApplySubscriptions(messageTypesHandled);
+                    return new ApplySubscriptions(messageTypesHandled, type => Task.FromResult(true));
```

Comment:
Not really related to this PR. But the AutoSubscribe feature is way to complex, envies way to many things it doesn't own and also reimplements some of the scanning logic. It feels very wrong. Can we do something here in another PR? @janovesk @SzymonPobiega